.github/workflows/release.yaml

```diff
@@ -0,0 +1,69 @@
+name: NSC client tool release
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
so nsc release tags do not have a `v` on them - not sure if this is an issue

Reply:
How would we tell apart "this tag is for a release" from others?
Any reason to not start using a v in future?

Reply:
For the record here: because it will break the self-updater logic.  I added comments to this effect in the update commit.